fbinternal/cloud/go/services/testcontroller/statemachines/enodebd.go

```
@@ -335,13 +444,23 @@ func trafficTest(trafficTestNumber int, stateNumber int, gatewayClient GatewayCl
 
 	resp, err := gatewayClient.GenerateTraffic(*config.NetworkID, trafficGWID, config.Ssid, config.SsidPw)
 	if resp == nil || err != nil {
+		if successState == subscriberActiveState {
+			pretext = fmt.Sprintf(trafficInactiveSubPretextFmt, *config.SubscriberID, *config.EnodebSN, *config.AgwConfig.TargetGatewayID, "SUCCEEDED")
+			postToSlack(machine.client, *config.AgwConfig.SLACKWebhook, false, pretext, fallback, "", "")
+			return successState, 1 * time.Minute, err
```

I don't think we should return the error here unless it's unexpected